ru/turing/courses/lesson6/Karashtina/ddl.sql

```diff
@@ -0,0 +1,118 @@
+-- Database generated with pgModeler (PostgreSQL Database Modeler).
+-- pgModeler version: 1.0.4
+-- PostgreSQL version: 15.0
+-- Project Site: pgmodeler.io
+-- Model Author: ---
+
+-- Database creation must be performed outside a multi lined SQL file.
+-- These commands were put in this file only as a convenience.
+--
+-- object: blog | type: DATABASE --
+
+
+-- object: public."user" | type: TABLE --
+-- DROP TABLE IF EXISTS public."user" CASCADE;
+CREATE TABLE public."user" (
+                               id bigint NOT NULL,
+                               avatar bytea,
+                               description varchar(250),
+                               username varchar(100) NOT NULL,
+                               full_name varchar(250),
+                               email varchar(250) NOT NULL,
+                               password varchar(200) NOT NULL,
+                               birth_date date,
+                               CONSTRAINT id_pk PRIMARY KEY (id) );
+-- ddl-end --
+ALTER TABLE public."user" OWNER TO postgres;
+-- ddl-end --
+
+-- object: public.article | type: TABLE --
+-- DROP TABLE IF EXISTS public.article CASCADE;
+CREATE TABLE public.article (
+                                id bigint NOT NULL GENERATED ALWAYS AS IDENTITY ,
+                                id_user bigint,
```

Comment:
Лучше уж
```suggestion
                                id_author bigint,
```
чтобы понятнее было